platform-workflow-module/src/test/java/com/softicar/platform/workflow/module/workflow/transition/WorkflowTransitionActionExecutorTest.java

```diff
@@ -0,0 +1,155 @@
+package com.softicar.platform.workflow.module.workflow.transition;
+
+import com.softicar.platform.core.module.user.AGUser;
+import com.softicar.platform.workflow.module.test.AbstractTestObjectWorkflowTest;
+import com.softicar.platform.workflow.module.test.WorkflowTestObject;
+import com.softicar.platform.workflow.module.test.WorkflowTestObjectTable;
+import com.softicar.platform.workflow.module.workflow.WorkflowNodeTestPrecondition;
+import com.softicar.platform.workflow.module.workflow.item.AGWorkflowItem;
+import com.softicar.platform.workflow.module.workflow.node.AGWorkflowNode;
+import com.softicar.platform.workflow.module.workflow.task.AGWorkflowTask;
+import com.softicar.platform.workflow.module.workflow.transition.execution.AGWorkflowTransitionExecution;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class WorkflowTransitionActionExecutorTest extends AbstractTestObjectWorkflowTest {
```

Comment:
We should consider renaming/aliasing the nodes employed in this test class:
- `rootNode` -> alias `sourceNode`, i.e. create a field with that name
- `nextNode` -> rename to `targetNode`

IMO that would make test cases (assertions, in particular) easier to comprehend.

Reply:
ok